app/views/components/_header.html.erb

```diff
@@ -1,13 +1,13 @@
 <header class="sdn-header sdn-header-950" role="banner" data-module="sdn-header">
   <div class="sdn-header__container govuk-width-container">
     <%= link_to root_path, class: 'sdn-header__link' do %>
-      <span class="sdn-header__product-name">Návody.Digital</span>
+      <%= image_tag 'logo.svg' %>
     <% end %>
     <div class="sdn-header__content">
       <button type="button" role="button" class="sdn-header__menu-button js-header-toggle" aria-controls="navigation" aria-label="Show or hide Top Level Navigation">Menu</button>
       <nav>
         <ul id="navigation" class="sdn-header__navigation" aria-label="Navigácia">
-          <% unless current_page?(root_path) || current_page?(search_path) %>
+          <% unless current_page?(root_path) || current_page?(search_path) || (params[:id] && current_page?(category_path)) %>
```

Comment:
Toto uz nejako boptna, dajme to asi do nejakeho helpera `show_big_search_bar?` a tam to schovajme

Reply:
spravil som